DOCUMENTATION.md

```diff
@@ -250,6 +250,75 @@ If you omit this argument your default email reply-to address will be set for th
 
 </details>
 
+### Send a document by email
+Send files without the need for email attachments.
+
+To send a document by email, add a placeholder field to the template then upload a file. The placeholder field will contain a secure link to download the document.
+
+[Contact the GOV.UK Notify team](https://www.notifications.service.gov.uk/support) to enable this function for your service.
+
+#### Add a placeholder field to the template
+
+In Notify, use double brackets to add a placeholder field to the email template. For example:
+
+"Download your document at: ((link_to_document))"
+
+
+#### Upload your document
+˜
+The document you upload must be a PDF file smaller than 2MB.
+
+Pass the file object as a value into the personalisation argument. For example:
+
+```javascript
+var fs = require('fs');
+
+fs.readFile('path/to/document.pdf', function(err, document) {
+	console.log(err);
+	notifyClient.sendEmail('your_notify_api_key', 'recipient@email.com', {
+		personalisation: notifyClient.prepareUpload(document),
```

Comment:
This needs a couple of clean-ups - first arg is templateId, not api key, and personalisation should be a dictionary with the key `link_to_document`. We should keep it the same as the python client docs for consistency. So something like

```js
notifyClient.sendEmail(templateId, emailAddress, {
    personalisation: {
        first_name: 'Amala',
        application_date: '2018-01-01',
        link_to_document: notifyClient.prepareUpload(document)
    }
}
```

Reply:
Ahh, good call, I will change it.